src/components/Container/Container.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import Movie from '../Movie/Movie.js';
+
+const Container = props => {
+  const movieComponents = props.movies.map((movie, index) => (
+    <Movie
+      key={index}
+      poster_path={movie.poster_path}
+      title={movie.title}
+      average_rating={movie.average_rating}
+    />
+  ));
+
+  return (
+    <section className='movie-grid'>
+      {props.movies.length ? movieComponents :
+        <p>Loading...</p>}
+    </section>
```

Comment:
Nice! Eventually this will have to conditionally render our currentMovie so that when a user clicks on a movie, this same container component is rendering that. So maybe we wrap this section in a React Fragment, that way we can render the movie-grid section or the individual movie section that we are building next.